flexget/plugins/input/imdb_list.py

```diff
@@ -54,8 +57,9 @@ def on_task_input(self, task, config):
         else:
             url = 'http://www.imdb.com/list/%s' % config['list']
 
-        log.debug('Requesting: %s' % url)
-        page = task.requests.get(url, params=params)
+        headers = {'Accept-Language': config.get('force_language')}
```

Comment:
This very much works. Tested with several languages. 
I'll look into adding this into imdb_lookup instead (probably should be in util though)